.circleci/config.yml

```diff
@@ -9,23 +9,22 @@ jobs:
           keys:
             - node-cache-{{ checksum "package-lock.json" }}
             - node-cache
```

Comment:
I think it's safe to remove this key

Reply:
It's a fallback key, recommended best practice from other circleci configs I've seen.

Reply:
there's no save step with this key if you want the fallback there should be something to fallback to, so we either remove it or add a save step with it